sagemaker-python-sdk/mxnet_mnist/mxnet_mnist_elastic_inference_local.ipynb

```diff
@@ -80,7 +80,7 @@
    "source": [
```

Comment:
<p>replace this part by some sample data. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>